src/main/java/robot/subsystems/drivetrain/pure_pursuit/Constants.java

```diff
@@ -3,7 +3,7 @@
 public class Constants {
     //DRIVING CONSTANTS
     public final static double LOOKAHEAD_DISTANCE = 0.3; //in meters
-    final static double TRACK_WIDTH = 3;
+    public final static double ROBOT_WIDTH = 0.6; //the distance between the left and right wheels on the robot
```

Comment:
Consider moving `ROBOT_WIDTH` into [`drivetrain/Constants.java`](https://github.com/DanKatzuv/BaseBot/blob/a86a22f234f81dbc960091207bc5fadac0e40751/src/main/java/robot/subsystems/drivetrain/Constants.java), it seems more reasonable to me.